### PR TITLE
If more than one yaxis in a jqplot graph, make sure there is space on right for it.

### LINKED ISSUE #13488: Dashboard, Widget evolution graph, Legend displayed outside graph
When 2 different metrics are displayed (with a different unit), then the second unit is displayed outside the widget:

bug graph

For legend to be readable, it needs to be fully inside the widget.

### PR DESCRIPTION
**Pull request by diosmosis**
evolution graph:

image

bar graph:

image

Fixes #13488

### CODE REVIEW

File: plugins/CoreVisualizations/javascripts/jqplot.js [hunk 1]
```diff
@@ -471,11 +471,39 @@
...
+            }
+
+            $tempAxisElement.remove();
+
+            function getAxisWidth(axis) {
+                var maxWidth = 0;
+                axis.ticks.forEach(function (tick) {
+                    var tickFormatted = $.jqplot.NumberFormatter(axis.tickOptions.formatString || '%s', tick);
+                    $tempAxisElement.find('span').text(tickFormatted);
+                    console.log(tickFormatted, $tempAxisElement.width());
```
Comment (sgiehl) [inline plugins/CoreVisualizations/javascripts/jqplot.js]: debug statement should be removed

**Review by sgiehl — CHANGES_REQUESTED** [review]
besides the debug statement, looks good to merge

**Comment by diosmosis** [comment]
Can't merge yet, some regressions in UI tests.

**Review by sgiehl — APPROVED** [review]

**Comment by sgiehl** [comment]
Guess https://builds-artifacts.matomo.org/matomo-org/matomo/13488-right-legend/32902/EvolutionGraph_bounce_rate.png needs an update. All other UI failures seem unrelated

**Pull request merged**